libcxx/include/__exception/exception_ptr.h

```diff
@@ -37,14 +39,16 @@ _LIBCPP_OVERRIDABLE_FUNC_VIS __cxa_exception* __cxa_init_primary_exception(
     void*,
     std::type_info*,
     void(
-#  if defined(_WIN32)
+#    if defined(_WIN32)
         __thiscall
-#  endif
+#    endif
             *)(void*)) throw();
```

Comment:
Pre-existing, no change requested: Should we replace these `throw()`'s with `_NOEXCEPT`? (perhaps in another PR)

Reply:
I think we can do it in this PR, I can't see a way for a potential need to revert just `throw() -> _NOEXCEPT`.
Should I?

Reply:
I would keep the patch as-is just to separate concerns.